test/only.js

```diff
@@ -0,0 +1,13 @@
+QUnit.module( "QUnit.only" );
+
+QUnit.test( "implicitly skipped test", function( assert ) {
+	assert.ok( false, "test should be skipped" );
+});
+
+QUnit.only( "only run this test", function( assert ) {
+	assert.ok( true, "only this test should run" );
+});
+
+QUnit.test( "another implicitly skipped test", function( assert ) {
+	assert.ok( false, "test should be skipped" );
+});
```

Comment:
That would work for me as well. It feels a little clever though and would limit something like a change to cumulative onlys. I'm good either way. I think I'd personally prefer the config over the ref swap, but I'm game for whichever you two think would be more appropriate.